indico_toolkit/indico_wrapper/modelop.py

```diff
@@ -0,0 +1,147 @@
+from indico import IndicoClient
```

Comment:
This is part of a separate PR-- shouldn't be here

Reply:
same thing with the associated test... that PR has changes that need to be resolved, this PR should have 0 conflicts with that PR